dev/cnf/oss_dependencies.maven

```diff
@@ -64,6 +64,10 @@ com.sun.faces:jsf-api:2.2.14
 com.sun.faces:jsf-impl:2.2.14
 com.sun.istack:istack-commons-runtime:4.0.0-M3
 com.sun.istack:istack-commons-tools:4.0.0-M3
+com.sun.mail:jakarta.mail:2.0.0
+com.sun.mail:jakarta.mail:2.0.0-RC5
+com.sun.mail:javax.mail:1.5.6
+com.sun.mail:javax.mail:1.6.2
```

Comment:
@uberskigeek Why two versions here? I know you've been trying to get this merged in for a while so I'll let this go if you open a follow up issue to sort it out.